designs/systems/index.md

```diff
@@ -0,0 +1,128 @@
+
+The Ownership card has the username of the system owner and a button to change the owner. Right after there is an editable
+list of users, besides the owner, who get notifications for this system.
+
+The Pools card has a list of pools this system belongs to, with a button to edit the list.
+
+The Excluded Families card has only one button, leading to the Excluded Families configuration (if done the way it is
+today, doesnt fit in a card.)
+
+The main Beaker Settings can be defined in the appropriate card. Type and architectures supported by the system can be 
+defined there, as well as the Kernel type if it is an ARM system. It also has the Lab Controller the system is attached 
+to. 
+
+There is a Condition dropdown which defines the current status for this system (Automated, Manual or Broken). If the status
+for a system is changed, the user needs to write down why it is changing. The text and timestamp will be displayed as the
+Last Condition Report. Beaker itself can change this status and update this text.
+
+The last card is Remove from Beaker. It has a button to completely remove the system from Beaker.
+
+![](img/system_details_settings.png)
+
+## System details - Power Options
+
+When a user clicks "Power Options" in the horizontal menu, all the information and configuration related to power is
+shown in the screen. 
+
+The "Power Control" card will tell if the system power can be controlled remotely and will have buttons to power the
+system on and off, reboot it, interrupt and clear netboot.
+
+A table with the 5 most recent power commands is also shown. There is a button for the user to see all of the power
+history for the machine. The table has 5 fields:
+- Date / Time: timestamp for the last status update for this power command
+- Trigger: identifies how the power command was issued
+- Status: Current status of the command
+- Action: the command itself
+- Message (optional): message returned by the command call, if any
+
+The "Power Settings" card will let the user set the power configuration for the system, as address, protocol, ID, username,
+password, etc.
+
+![](img/system_details_power.png)
+
+## System details - Provisioning
+
+When the "Provisioning" option is selected in the horizontal menu, there will be two cards in the screen.
+
+One of the cards will have the options to provision the system with a specific distro from the library. The user will be
+able to select the family, tag, distro and tree as well as a set of Install Options to provision the system. A checkbox
+may be selected to reboot the system.
+
+Install Options can be defined in the second card. The "default" set of options will always be available, for all families
+and architectures, with specific KS metadata and Kernel options for the installation. The user may click the "Add" button
+to create customized sets of options for specific families and / or architectures supported by the system, and edit them
```

Comment:
Is there any way for user to see configuration for specific families/architectures on the screen?

Reply:
Yes. The "Special Option Set" defined there is for Fedora 32 and x86_64, for instance. My main concern here is the scrolling bar when there is a lot of option sets...